lib/wopinator/discovery.rb

```diff
@@ -104,6 +107,24 @@ def parse_url(url)
       Addressable::URI.parse(url.gsub(/&#38;/, '&').gsub(/<.*?=.*?>/, '').gsub(/(\?|&)$/, ''))
     end
 
+    def filter_action_query_params(url, params)
+      return {} unless params.any?
+
+      supported = extract_supported_action_query_params(url)
+
+      {}.tap do |hash|
+        params.each do |k, v|
+          if key = supported[k]
+            hash[key] = v
+          end
+        end
+      end
+    end
+
+    def extract_supported_action_query_params(url)
```

Comment:
This is then used to add back to the URL things like `IsLicensedUser=1` by replacing BUSINESS_USER with `1`.